server/protocols/protocols-library/src/test/resources/testServerDefaults.xml

```diff
@@ -0,0 +1,2 @@
+<testerver enabled="true">
```

Comment:
```suggestion
<testserver enabled="true">
```

Idem for all the other xml files :)

Reply:
Sigh, copy and paste strikes again. Thanks for spotting this.